pkg/agent/nodeportlocal/k8s/annotations.go

```diff
@@ -84,6 +85,14 @@ func assignPodAnnotation(pod *corev1.Pod, nodeIP string, containerPort, nodePort
 	pod.Annotations = current
 }
 
+func removePodAnnotation(pod *corev1.Pod) {
+	current := pod.Annotations
+
+	klog.V(2).Infof("Removing annotation from pod: %v", pod.Name)
+	delete(current, NPLAnnotationStr)
+	pod.Annotations = current
```

Comment:
Since `pod.Annotations` is not too long, maybe just `delete(pod.Annotations, NPLAnnotationStr) to avoid the extra variable and lines?

Reply:
Done, thanks